lib/LoginPage/loginPage.dart

```diff
@@ -35,49 +35,43 @@ class _LoginPageState extends State<LoginPage> {
                 0.5,
                 Row(
                   mainAxisAlignment: MainAxisAlignment.center,
-                  children: <Widget>[
-                    FlatButton(
-                      highlightColor: Colors.transparent,
-                      splashColor: Colors.transparent,
-                      color: _pageLogin
-                          ? Color.fromRGBO(143, 148, 251, 1)
-                          : Colors.transparent,
-                      child: Text(
-                        "Login",
-                        style: TextStyle(
-                          color: _pageLogin
-                              ? Colors.white
-                              : Color.fromRGBO(143, 148, 251, 1),
-                        ),
-                      ),
-                      onPressed: () {
-                        _togglePage(true);
-                      },
-                    ),
-                    FlatButton(
-                      highlightColor: Colors.transparent,
-                      splashColor: Colors.transparent,
-                      color: _pageLogin
-                          ? Colors.transparent
-                          : Color.fromRGBO(143, 148, 251, 1),
-                      child: Text(
-                        "SignUp",
-                        style: TextStyle(
-                          color: _pageLogin
-                              ? Color.fromRGBO(143, 148, 251, 1)
-                              : Colors.white,
-                        ),
-                      ),
-                      onPressed: () {
-                        _togglePage(false);
-                      },
-                    ),
-                  ],
+                  children: <Widget>[ElevatedButton(
+  style: ElevatedButton.styleFrom(
+    foregroundColor: _pageLogin
+        ? Colors.white
+        : const Color.fromRGBO(143, 148, 251, 1), backgroundColor: _pageLogin
+        ? const Color.fromRGBO(143, 148, 251, 1)
+        : Colors.transparent,
+    elevation: 0, // Adjust elevation as needed
```

Comment:
You are repeating `elevation` and `tapTargetSize` property frequently. So you can define a seperate variable for `ButtonStyle` and override the necessary values using `copywith`. 

Defining variable:

```
late final ButtonStyle _buttonStyle = ElevatedButton.styleFrom(
    elevation: 0,
    tapTargetSize: MaterialTapTargetSize.shrinkWrap,
  );
```